smdebug/core/logger.py

```diff
@@ -60,6 +60,16 @@ def get_logger(name="smdebug"):
             datefmt="%Y-%m-%d %H:%M:%S",
         )
 
+        if "SM_TRAINING_ENV" in os.environ:
```

Comment:
use this function: https://github.com/awslabs/sagemaker-debugger/blob/master/smdebug/core/sagemaker_utils.py#L5